apps/betterangels-backend/accounts/schema.py

```diff
@@ -241,17 +253,9 @@ def update_client_profile(self, info: Info, data: UpdateClientProfileInput) -> C
                         household_member_updates_by_id[str(household_member.id)],
                     )
 
-            client_profile = resolvers.update(
-                info,
-                client_profile,
-                {
-                    **client_profile_data,
-                },
-            )
-
             if hmis_profiles:
                 hmis_profile_updates_by_id = {hp["id"]: hp for hp in hmis_profiles if hp.get("id")}
-                hmis_profiles_to_create = [hp for hp in hmis_profiles if hp.get("id") is None]
+                hmis_profiles_to_create = [hp for hp in hmis_profiles if not hp.get("id")]
```

Comment:
for posterity: this was a substantive change, not just style/pattern. When `.get()` doesn't get anything, it returns falsey, but not `None`